api.js

```diff
@@ -87,8 +86,8 @@ API.promptDefaults = function(_useDefaults) {
  *
  */
 API.install = function(name, target, options) {
-  return install.install.apply(install, arguments);
-}
+  return install.install.apply(install, [name, target, options]);
```

Comment:
Yes a direct call would be nicer for these. Luckily there are no checks of `arguments.length` in install, as it does take variable argument numbers.